docs/infrastructure/microk8s-quickstart.md

```diff
@@ -83,10 +83,13 @@ Now that MicroK8s is running on the machine we can proceed to enabling the neces
     microk8s.config > ~/.kube/config
     ```
 
-5.  Install Helm vervsion 3.2.0:
+5.  Install Helm version 3.2.1:
 
     ```bash
-    sudo snap install helm --classic --channel=3.2/stable
+    curl -L "https://get.helm.sh/helm-v3.2.1-linux-amd64.tar.gz" | tar -zxf - &&
+    mv linux-amd64/helm /usr/local/bin/helm &&
+    chmod +x /usr/local/bin/helm &&
+    rm -r linux-amd64
```

Comment:
@machadoit, please validate that this is the correct version that we want customers to install and use.

Reply:
Thx! :D

Reply:
Just needed a couple sudos